packages/connect/extension-resolver.cjs

```diff
@@ -0,0 +1,29 @@
+let defaultResolver;
+
+function requireDefaultResolver() {
+  if (!defaultResolver) {
+    try {
+      defaultResolver = import (`jest-resolve/build/defaultResolver`);
+    } catch (error) {
+      defaultResolver = import (`jest-resolve/build/default_resolver`);
+    }
+  }
+
+  return defaultResolver;
+}
+
+
+module.exports = (request, options) => {
+  const {basedir, defaultResolver, extensions} = options;
+
+  if (!defaultResolver) {
+    defaultResolver = requireDefaultResolver();
+  }
+
+  try {
+    return defaultResolver(request, options);
+  } catch (e) {
+    console.log(`${e} - Temporary replace extension '.js' to '.ts' for tests to run.\n`)
```

Comment:
Check `e.code` and throw the error if it is not `MODULE_NOT_FOUND` also check that request ends with `'js'` ... we should only be trying to fix the resolve in those cases otherwise we should throw early